source/Ship.cpp

```diff
@@ -4013,8 +4030,11 @@ void Ship::ExpendAmmo(const Weapon &weapon)
 		heat -= weapon.AmmoUsage() * .5 * ammo->Mass() * MAXIMUM_TEMPERATURE * Heat();
 		AddOutfit(ammo, -weapon.AmmoUsage());
 		// Only the player's ships make use of attraction and deterrence.
-		if(isYours && !OutfitCount(ammo) && ammo->AmmoUsage())
-			deterrence = CalculateDeterrence();
+		if(!OutfitCount(ammo) && ammo->AmmoUsage())
+		{
+			if(isYours)
+				deterrence = CalculateDeterrence();
+		}
```

Comment:
This can revert to what it was before now.
```suggestion
		if(isYours && !OutfitCount(ammo) && ammo->AmmoUsage())
			deterrence = CalculateDeterrence();
```